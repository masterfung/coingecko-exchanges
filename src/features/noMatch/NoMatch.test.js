import { shallow } from "enzyme";
import NoMatch from "./NoMatch";

describe("NoMatch Component", () => {
  it("should render NoMatch component", () => {
    expect(shallow(<NoMatch />)).toMatchSnapshot();
  });
});