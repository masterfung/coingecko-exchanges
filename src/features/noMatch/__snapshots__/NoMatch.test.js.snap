// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch Component should render NoMatch component 1`] = `ShallowWrapper {}`;
